### ISSUE TITLE
cf revisions - show which revision is deployable true/false

### ISSUE THREAD

**Issue by danfein** [cloudfoundry/cli#1949]
**Please provide details on the following items. Failure to do so may result in deletion of your feature request.** 

**What's the user value of this feature request?**
The current `cf revisions` command doesn’t tell the user which revision can be rolled back to.
Adding a `deployable` boolean column to the output will be a convenient signal to users regarding which revisions can even be considered for roll-back/forward.

**Who is the functionality for?**
Cf cli users who may be planning to rollback to a prior revision

**How often will this functionality be used by the user?**
How often will they use this feature?
It will be used whenever a user needs to roll-back to a previous version of their app --- this would typically be during a time of criticality and potential high-stress.

**Who else is affected by the change?**
Does this feature cause a breaking change? Are other users affected by this change?
This is not a breaking change and it’s a change to a feature 

**Is your feature request related to a problem? Please describe.**
When viewing the list of past revisions, it is not currently possible to see which revisions still have a droplet on disk. Attempting to rollback to a revision with an expired droplet can lead to a bit of guess and check and is frustrating 

**Describe the solution you'd like**
There may be a number of reasons a past revision is not deployable, including the status of the underlying droplet. I propose a new column in the revisions table to make it easier to identify valid rollback choices from the list.

**Describe alternatives you've considered**
We considered a flag to only show deployable choices, or to by default only show deployable choices, and to have a flag to show all past revisions, which felt like it risked hiding important information.

We also considered a column for droplet status, but in testing it was not always clear to users what the consequences of the droplet status were. 

**Additional context**

```
$ cf revisions testing-rolling
Getting revisions for app testing-rolling in org meehp / space maahp as admin...
OK

version   guid                                   description            deployable    created at 
1         5764513a-2593-44a3-8a8e-5e0d47b60ccc   initial revision.      true          2020-03-04T13:12:21Z
2         7353351e-9de6-4463-95f4-9ec198bf80fc   new droplet deployed.  true          2020-03-04T13:16:46Z
3         493f20f3-74ef-4da7-bd3c-2c906db6f711   new droplet deployed.  false         2020-03-04T12:16:46Z
```

**Comment by heyjcollins** [comment]
@simonjohansson - we're beginning our tract of work to fully flesh out the revisions feature and thought you might be interested in the first of several enhancements we are proposing. 

This issue is being created as a vehicle for feedback before we move toward implementation.

Thanks for your previous contribution. Looking forward to working with you further.

Cheers,

Josh and @danfein

**Comment by heyjcollins** [comment]
related to #1908

**Comment by danfein** [comment]
@pivotalrobbie

**Comment by simonjohansson** [comment]
@heyjcollins wonderful, please share a Tracker link :)

Would be interested in contributing more if possible.

**Comment by danfein** [comment]
Hello hello Simon,
Autogenerated tracker link https://www.pivotaltracker.com/n/projects/892938/stories/173183537
Josh could answer this better, but I suspect we will fill this out more as we get closer to picking it up.

Thanks for taking a look at revisions, I am glad to see it make it into the CLI

**Comment by simonjohansson** [comment]
@danfein thanks, I was more curious if there was a public track/epic around the whole rolling/rollback/revisions based workflow.

**Comment by danfein** [comment]
@simonjohansson Ahh, thanks for the clarification, If it helps, there is a revisions tag that has the stories we have in the backlog around getting revisions capabilities surfaced in the CLI. 
I didn't see an epic defined in Aha! (the other place we track work).

https://www.pivotaltracker.com/n/projects/892938/search?q=label%3A%22revisions%22

**Comment by reid47** [comment]
We recently released [v7.2.0](https://github.com/cloudfoundry/cli/releases/tag/v7.2.0), which further improves the `revisions` command. Thanks for your help!
